wagtail/admin/static_src/wagtailadmin/scss/layouts/account.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../../../client/scss/settings/variables';
+
+.top-padding {
```

Comment:
I avoided putting this into the main CSS because it feels like a hacky workaround. Ideally, the salmon bars should allow space for the content beneath.